<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Manish" failures="2" tests="2" name="Test(failed)(failed)" time="0.083" errors="0" timestamp="2023-11-04T00:32:02 IST">
  <testcase classname="FailedTestcase.Testcase1" name="test3" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at FailedTestcase.Testcase1.test3(Testcase1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="FailedTestcase.Testcase1" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at FailedTestcase.Testcase1.test1(Testcase1.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="FailedTestcase.Testcase1" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test(failed)(failed) -->
